### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by c-benoit** [getlift/lift#352] [General]
-

**Comment by mnapoli** [comment]
Hey! 

The idea of that construct is to have everything behind 1 domain name, so you can't set up a domain just for S3 assets.

Is there a reason you want to do that?

**Reply by c-benoit** [reply]
*Edit: I made an error by confusing two buckets. I'm not using Lift for the one I'm having issues with 😅
Apologies for the confusion and the topic
